examples/compare_connectivity_over_time_over_trial.py

```diff
@@ -0,0 +1,427 @@
+"""
+=====================================================================
+Comparing spectral connectivity computed over time or over trials
+=====================================================================
+
+This example demonstrates the difference between spectral connectivity
+computed over time or over trials.
+
+A brief background on the difference between the two conditions is provided,
+followed by examples on simulated data and real EEG data.
+
+"""
+###############################################################################
+# Background
+# ----------
+#
+# Spectral connectivity is a method for inferring the relationship between
+# channels decomposed at different frequencies of interest. The channels
+# could be M/EEG sensors or brain regions estimated with source localization.
+#
+# There are multiple different spectral connectivity measures, e.g. coherence,
+# imaginary part of coherence, phase locking value, and envelope correlations.
+# Additionally, there are also multiple methods to estimate the frequency
+# content, e.g. Fourier transforms, Morlet wavelets and multitapers.
+#
+# In this example, we focus on the two functions
+# :func:`mne_connectivity.spectral_connectivity_epochs` and
+# :func:`mne_connectivity.spectral_connectivity_time`,
+# which both computes connectivity from epoched data :class:`mne.Epochs`.
+#
+# Both functions contain the options to choose the connectivity measurements
+# of interest and the method to decompose the frequency content.
+# The crucial difference between when to use which function lies on
+# the experimental setup and type of tasks.
+#
+# If the data is obtained for repeated trials of a time-locked event,
+# e.g. ERP data, then :func:`mne_connectivity.spectral_connectivity_epochs`
```

Comment:
Not everyone that I know of that works w/ mne is familiar w/ ERP data. Perhaps we can link either another MNE-python tutorial/example/glossary, or a paper that defines it.